### ISSUE TITLE
HDMI or USB-C DP out

### ISSUE THREAD

**Issue by koluris** [dkoluris/ZEPHYRUS-M15-GU502-OpenCore#2]
Since the dGPU is disabled to preserve power, we need to investigate whether HDMI or USB-C DP out is connected to the iGPU. My best guess is that none is, but we need to make sure.

**Comment by Alex-V2000** [comment]
Man, go in Nvidia Control Panel - PhysX and upload here a screenshoot !

**Comment by koluris** [comment]
![screen](https://user-images.githubusercontent.com/31432553/106136295-c341cc00-6171-11eb-8cbb-b6c435bec511.jpg)

Sorry about the quality!

**Comment by Alex-V2000** [comment]
Yes, it's the same problem as on my laptop, the USB-C port and the HDMI port are connected to the NVIDIA card, therefore we can't see the image on macOS through these ports!

**Comment by Alex-V2000** [comment]
I tried all sorts of possibilities, but I didn't succeed in any way!

**Comment by Alex-V2000** [comment]
Only the LVDS connector is connected to the Intel board, which is connected to the laptop display

**Comment by koluris** [comment]
Tough luck then no problem! This is the main reason I am not using this laptop as my main iOS development rig. Thanks for checking.

**Comment by Alex-V2000** [comment]
Hello! I know it has nothing to do with this problem but I want to ask you, on your laptop, read the RPM of the fan from the processor?

**Comment by koluris** [comment]
Hmmm, where can I check this? @Alex-V2000

**Comment by Alex-V2000** [comment]
With iStats Menus or https://github.com/CloverHackyColor/HWMonitorSMC2/releases

**Comment by koluris** [comment]
Hmm i tried using HWMonitorSMC2 and it reports everything except to show me the fan speeds...

**Comment by Alex-V2000** [comment]
Yes, I have the same problem, but I would like to make him read the RPM and be able to adjust it to Full Blast or Auto 😢

**Comment by koluris** [comment]
I thought that SMCSuperIO was the kext that passes the RPM speed to the system, it seems that it doesn't work. I will look it up.

**Comment by Alex-V2000** [comment]
Thanks man, if you manage to do something, I will try again and maybe I can find a solution!

**Comment by alllexx88** [comment]
> Tough luck then no problem! This is the main reason I am not using this laptop as my main iOS development rig. Thanks for checking.

As a workaround, you could try some DisplayLink device to connect (an) external display(s). I used a cheap one from aliexpress to connect a second FHD display to a laptop with only a single HDMI port and no other external display connectivity. There're also more advanced options available, especially if Thunderbolt 3 port works.

**Comment by koluris** [comment]
> > Tough luck then no problem! This is the main reason I am not using this laptop as my main iOS development rig. Thanks for checking.
> 
> As a workaround, you could try some DisplayLink device to connect (an) external display(s). I used a cheap one from aliexpress to connect a second FHD display to a laptop with only a single HDMI port and no other external display connectivity. There're also more advanced options available, especially if Thunderbolt 3 port works.

I had this in mind until I read about this kind of solution using CPU power to transmit the actual image over USB 3.1, and the fact that video motion or fast gaming pace has problems even with a DisplayLink solution. Nonetheless, I cannot find a DisplayLink device that will transmit 4k@60hz using a USB 3.1 port.

**Comment by alllexx88** [comment]
> I cannot find a DisplayLink device that will transmit 4k@60hz using a USB 3.1 port.

Well, they claim it's possible to have **4** 4k@60Hz external displays on one USB 3 Gen 1 with their D-6xxx chipsets:  https://www.youtube.com/watch?v=eimqrJ_G2Xo
And here's an example of 2x4K@60Hz from Amazon: https://www.amazon.com/StarTech-USB-Dual-DisplayPort-Adapter/dp/B07C69HG33/ref=sr_1_1_sspa (though far from being cheap).

> CPU power to transmit the actual image

Indeed, I've read some reports stating 4k videos on YouTube utilize up to 50% CPU, but I don't know their setup. Wonder how 10750H would handle it, I assume it wouldn't even sweat :)

**Comment by koluris** [comment]
> > I cannot find a DisplayLink device that will transmit 4k@60hz using a USB 3.1 port.
> 
> Well, they claim it's possible to have **4** 4k@60Hz external displays on one USB 3 Gen 1 with their D-6xxx chipsets: https://www.youtube.com/watch?v=eimqrJ_G2Xo
> And here's an example of 2x4K@60Hz from Amazon: https://www.amazon.com/StarTech-USB-Dual-DisplayPort-Adapter/dp/B07C69HG33/ref=sr_1_1_sspa (though far from being cheap).
> 
> > CPU power to transmit the actual image
> 
> Indeed, I've read some reports stating 4k videos on YouTube utilize up to 50% CPU, but I don't know their setup. Wonder how 10750H would handle it, I assume it wouldn't even sweat :)

Not bad, I would actually try one of those if it was half price and would offer just one output 4k@60hz. I am reluctant to try one at 110 euros though :|

**Comment by alllexx88** [comment]
> 110 euros

I stumbled across this: https://www.amazon.com/StarTech-com-USBA2DPGB-USB-Dual-DisplayPort/dp/B076HTV3PR/ref=sr_1_4
It's about 70 euros + delivery (not sure how much is delivery to your country)
Can't find anything cheaper with 4k@60Hz

**Comment by koluris** [comment]
Thanks, wish there was just a simple, compact, cheap adapter USB -> HDMI 2.0 just to try it out you know
